python/change-notes/2021-02-23-port-insecure-default-protocol.md

```diff
@@ -0,0 +1,3 @@
+lgtm,codescanning
+* Ported query detecting insecure protocol creation from default values (`py/insecure-default-protocol`) to use the new API graphs. Since the query is somewhat out-dated, its scope has been limited to relevant versions of Python and
+the help file has been updated.
```

Comment:
I think your description in the PR summary was clearer, so I suggest:

```suggestion
* Changed the query that detects insecure protocol creation from default values (`py/insecure-default-protocol`) to use the new API graphs. Modern versions of Python include fluent APIs that change default values after construction, so the query now reports results only in versions of Python where fluent APIs are not available.
```

I don't think we need to highlight the fact that we've updated the help file to match the query behavior.